SPECS-EXTENDED/aspell/aspell.spec

```diff
@@ -1,25 +1,27 @@
-Summary: Spell checker
-Name: aspell
-Version: 0.60.8
-Release: 5%{?dist}
+Summary:        Spell checker
+Name:           aspell
+Version:        0.60.8
+Release:        6%{?dist}
 # LGPLv2+ .. common/gettext.h
 # LGPLv2  .. modules/speller/default/phonet.hpp,
 #            modules/speller/default/phonet.cpp,
 #            modules/speller/default/affix.cpp
 # GPLv2+  .. ltmain.sh, misc/po-filter.c
 # BSD     .. myspell/munch.c
-License: LGPLv2+ and LGPLv2 and GPLv2+ and BSD
+License:        LGPLv2+ AND LGPLv2 AND GPLv2+ AND BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL: http://aspell.net/
-Source: ftp://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
-
-Patch0: aspell-0.60.7-fileconflict.patch
-Patch1: aspell-0.60.7-pspell_conf.patch
-Patch2: aspell-0.60.7-mp.patch
-
-BuildRequires: gcc-c++
-BuildRequires: chrpath, gettext, ncurses-devel, pkgconfig, perl-interpreter
+URL:            http://aspell.net/
+Source:         https://github.com/GNUAspell/aspell/archive/rel-%{version}.tar.gz
```

Comment:
This tar's root directory is aspell-rel-0.60.8

You can check this by wget'ing the source and running tar tvf foo.tar.gz. 

To make this build, you will likely need to add a "-n %{name}-rel-%{version}" to the %setup macro in the %prep section of this spec